CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* PayPal
+    * Add optional property `PayPalCheckoutRequest.setUserAuthenticationEmail()`
+* Add ShopperInsights module (BETA)
```

Comment:
```suggestion
* ShopperInsights (BETA)
```
We should also probably add a description here of the new methods/how a merchant would use this. We can do something similar to what we did for PayPalMessaging below

Reply:
Sarah mentioned that this closely matches what iOS has in the changelog: https://github.com/braintree/braintree_android/pull/1006#discussion_r1608747071


Maybe adding this line back is enough?
```
  * Add `ShopperInsightsClient.getRecommendedPaymentMethods()` for returning recommendations based on the buyer
```

Reply:
Gotcha, yeah I get removing some of the complexity but I think we'd want to call out the public interfaces that we'd want merchants to use. Otherwise we just drop this feature with no details. Curious what @scannillo thinks as well.

Reply:
Makes sense! I'll add that bullet back.

Reply:
Added the bullet back in https://github.com/braintree/braintree_android/pull/1024